website/docs/admin_hosts_houdini.md

```diff
@@ -3,9 +3,26 @@ id: admin_hosts_houdini
 title: Houdini
 sidebar_label: Houdini
 ---
+## General Settings
+### JOB Path
+you can add your studios preffered JOB Path, JOB value will be checked and updated on file save and open.
+Disableing this option will effectivly turn off this feature.
+
+JOB Path can be:
+- Arbitrary hardcoded path
+- Openpype template path
+    > This allows dynamic values for assets or shots.<br>
+    > Using template keys is supported but formatting keys capitalization variants is not,
+    >   e.g. {Asset} and {ASSET} won't work
+- empty
+    > In this case, JOB will be synced to HIP
+
+![update job on context change](assets/houdini/update-job-context-change.png)
+
+
```

Comment:
Should we add a note that it creates the folder as well?

```suggestion
If the folder does not create on the context change it will be created by this feature so that `$JOB` will always try to point to an existing folder.
```

```